src/hooks/use-document-sync-to-firebase.ts

```diff
@@ -8,6 +9,11 @@ import { isPublishedType, LearningLogDocument, LearningLogPublication, PersonalD
          PersonalPublication, ProblemDocument, ProblemPublication, SupportPublication
         } from "../models/document/document-types";
 import { UserModelType } from "../models/stores/user";
+import { Firestore } from "src/lib/firestore";
+import { useMutation, UseMutationOptions } from "react-query";
+import { onSnapshot, SnapshotOut } from "@concord-consortium/mobx-state-tree";
```

Comment:
I think we are set up that this can just be imported as `...from "mobx-state-tree";`